spark/src/test/scala/org/apache/comet/parquet/ParquetReadSuite.scala

```diff
@@ -1460,6 +1460,59 @@ class ParquetReadV1Suite extends ParquetReadSuite with AdaptiveSparkPlanHelper {
           v1 = Some("parquet"))
     }
   }
+
+  test("test V1 parquet scan filter pushdown of primitive types uses native_iceberg_compat") {
```

Comment:
Can we enable this test for both native_datafusion and native_iceberg_compat?

Reply:
I enabled the test for native_datafusion, but the current native_datafusion mode always push down fitlers, I added a comment:

https://github.com/apache/datafusion-comet/blob/8896f67247a47ed6463f297102070e6f5ea9c0aa/spark/src/test/scala/org/apache/comet/parquet/ParquetReadSuite.scala#L1516-L1519